src/_content/posts/getting-to-know-folks.md

```diff
@@ -0,0 +1,266 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2025-07-13 12:00:00
+layout: post
+cover:
+    url: /assets/img/2025/preconference.png
+    alt: interviewees images
+title: "Getting to know folks: A pre-conference interview"
+---
+
+Hello Djangonauts!
+
+DjangoCon US is coming up, and our excitement is building. 
+
+We’re spending a little time getting to know people who will be attending DjangoCon US this September. We asked various people who have different connections and experiences with the Django community.
+
+- [Apoorv Garg](https://www.linkedin.com/in/apoorv-garg-/) is a first-time attendee at DjangoCon US
+- [Richel Agyemang](https://www.linkedin.com/in/richel-agyemang-468379211/) is a first-time organizer
+- [Katherine Michel](https://fosstodon.org/@kati) is a long-time attendee, speaker and organizer of DjangoCon US 
```

Comment:
Could this be change to Katherine "Kati" Michel and the subsequent references to me be changed to Kati? People call me both names, but I prefer Kati these days.